app/point.py

```diff
@@ -17,7 +17,7 @@ def __hash__(self) -> int:
         # which is NOT a best practice, but you will be able to predict
         # a hash value by coordinates of the point and its index
         # in the hashtable as well
-        return hash((self.x, self.y))
+        return int(self.x + self.y)
```

Comment:
Don't change Point class!

Reply:
I don't rememder I changed it. Should I replace int with hash?